src/types.ts

```diff
@@ -0,0 +1,34 @@
+import type { Message } from 'whatsapp-chat-parser';
+import JSZip from 'jszip';
+
+type FilterMode = 'index' | 'date';
+
+type ExtractedFile = string | JSZip | null;
+
+interface IndexedMessage extends Message {
+  index: number;
+}
+
+interface ILimits {
```

Comment:
I think it would be better having only one generic limits interface, instead of one for numbers and one for strings

Reply:
Looking much better now, gg @Pustur